python/rikai/contrib/torch/detections.py

```diff
@@ -14,4 +14,4 @@
 
 
 # Object detection output schema
-OUTPUT_SCHEMA = "array<struct<box:box2d, score:float, label_id:int>>"
+OUTPUT_SCHEMA = "array<struct<box:box2d, score:float, label:int>>"
```

Comment:
label_id is the id of label, label should be the text of the label

I think we should use label_id here.

Reply:
ok,